website/src/app/blog/apr-2024-update/_page.tsx

```diff
@@ -1,13 +1,15 @@
 "use client";
 import Post from "@/components/Blog/Post";
 import Content from "./readme.mdx";
+import gravatar from "@/lib/gravatar";
 
 export default function _Page() {
   return (
     <Post
       authorName="Jamil Bou Kheir"
       authorTitle="Founder"
       authorEmail="jamil@firezone.dev"
+      authorAvatarSrc={gravatar("jamil@firezone.dev")}
```

Comment:
To make the blogs not depend on Gravatar I had to refactor all these

Reply:
👍